src/bootstrap/clean.rs

```diff
@@ -76,9 +76,7 @@ fn rm_rf(path: &Path) {
                 fs::remove_dir(p).or_else(|e| {
                     // Check for dir not empty on Windows
                     #[cfg(windows)]
-                    if matches!(e.kind(), std::io::ErrorKind::Other)
-                        && e.raw_os_error() == Some(145)
-                    {
+                    if e.raw_os_error() == Some(145) {
```

Comment:
Can you add a FIXME here for replacement with [`ErrorKind::DirectoryNotEmpty`](https://doc.rust-lang.org/nightly/std/io/enum.ErrorKind.html#variant.DirectoryNotEmpty) once that stabilizes?